src/generator.cs

```diff
@@ -5981,6 +5983,15 @@ void WriteMarkDirtyIfDerived (StreamWriter sw, Type type)
 		}
 	}
 
+	public void PrintXpcInterfaceAttribute (ICustomAttributeProvider mi)
+	{
+		var p = AttributeManager.GetCustomAttribute<XpcInterfaceAttribute> (mi);
```

Comment:
If possible, use `AttributeManager.HasAttribute<XpcInterfaceAttribute> (mi)` instead, since it can be faster.